acceptance/README.md

```diff
@@ -145,6 +145,33 @@ There also may be scenarios where you want to specify the host(s) to release. E.
     HOST_NAMES=lvwwr9tdplg351u bundle exec rake ci:release_hosts
     HOST_NAMES=lvwwr9tdplg351u,ylrqjh5l6xvym4t bundle exec rake ci:release_hosts
 
+### Running tagged tests
+
+Tests runs can be segmented by tags defined in the tests themselves. These
+tags have been standardized into the following.
+
+* risk:high
+* risk:medium
+* risk:low
```

Comment:
Hrmmm.... I would argue that untagged tests are high risk as;
1 - Having no tag (probably) shows no thought has gone into how risky they are, therefore we should run them as often as possible to figure out how risky they really are
2  - It's far too easy to forget to tag a test and then only find out  many days later that it failed